diff --git a/types/index.d.ts b/types/index.d.ts
index 668f67e55d1b4b2abe1f2c71d3783b85b8164b2f..897b1df30c12110b00dd86c54e09877871ab936d 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -27,7 +27,7 @@
 
 declare class NativeDate extends globalThis.Date { }
 
-declare module 'mongoose' {
+declare module 'mongoose-v8' {
   import Kareem = require('kareem');
   import events = require('events');
   import mongodb = require('mongodb');
